carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/Screenshot.java

```diff
@@ -296,7 +306,7 @@ private static String capture(WebDriver driver, boolean isTakeScreenshot, String
                 //hotfix to converting proxy into the valid driver
                 if (driver instanceof Proxy) {
     				try {
-        				InvocationHandler innerProxy = Proxy.getInvocationHandler(((Proxy) driver));
+        				InvocationHandler innerProxy = Proxy.getInvocationHandler((Proxy) driver);
         				// "arg$2" is by default RemoteWebDriver;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.53.207.202/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)